### DISCUSSION TITLE
Does anyone now how to integrate react-navigation in the library project?

### DISCUSSION THREAD

**Discussion by sharifme04** [callstack/react-native-builder-bob#295] [Q&A]
Current behavior
Hi Currently I am working on a react-native SDK for my work. But I am failed to integrate react-navigation (version 6 , also tried version 5) as it has some configuration setup for native. But in the library, I do not have those files(For android for example  `MainActivity.java`). I have used this to create my library https://reactnative.dev/docs/native-modules-setup.

Does anyone have any idea how to integrate `react-navigation `like a library in a react-native SDK or react-native library ?

![image](https://user-images.githubusercontent.com/24765222/145791620-a9c270ba-8be6-4baf-97ce-1408739e52a6.png)
![image](https://user-images.githubusercontent.com/24765222/145791684-3c6cbd13-9c89-4b3c-bff6-17b4124dbcf9.png)

**Comment by igorrmon** [comment]
Hi @sharifme04.
I have the same problem, did you manage to solve it?
